app/src/main/res/layout/activity_maps.xml

```diff
@@ -16,11 +16,11 @@ tools:context=".Activities.MapsActivity">
         android:elevation="4dp"
         android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
         app:popupTheme="@style/ThemeOverlay.AppCompat.Light"/>
-<!--    <fragment
-        android:id="@+id/autocomplete_fragment"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:name="com.google.android.libraries.places.widget.AutocompleteSupportFragment" />-->
+    <!--    <fragment
+            android:id="@+id/autocomplete_fragment"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:name="com.google.android.libraries.places.widget.AutocompleteSupportFragment" />-->
```

Comment:
This commented out code can be removed since we have modified the search bar and won't be using Google's

Reply:
ok will remove these lines